src/plugins/share/public/url_service/redirect/components/empty_prompt.tsx

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import * as React from 'react';
+
+import { EuiButtonEmpty } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+import { ChromeDocTitle } from '@kbn/core-chrome-browser';
+import { NotFoundPrompt } from '@kbn/shared-ux-prompt-not-found';
+
+const defaultTitle = i18n.translate('share.urlService.redirect.components.Error.title', {
+  defaultMessage: 'Unable to open URL',
+  description:
+    'Title displayed to user in redirect endpoint when redirection cannot be performed successfully.',
+});
+
+const defaultBody = i18n.translate('share.urlService.redirect.components.Error.body', {
+  defaultMessage:
+    'The stored URL for this link could not be opened. This could be due to lost Kibana Saved Object data.',
+});
+
+export interface ErrorProps {
+  title?: string;
+  body?: string;
+  docTitle: ChromeDocTitle;
+  error: Error;
+}
+
+export const RedirectEmptyPrompt: React.FC<ErrorProps> = ({
+  title = defaultTitle,
+  body = defaultBody,
+  docTitle,
+  error,
+}) => {
+  // eslint-disable-next-line no-console
+  console.error('Short URL Redirect Error', error);
+
+  // Using the current URL containing "/app/r/", make a URL to the root basePath
+  // by trimming that part to end up at the Home app or project home.
+  const currentUrl = window.location.href;
+  const newUrl = currentUrl.replace(/\/app\/r\/.*/, '');
```

Comment:
Why wouldn't we use home href from settings instead?

Reply:
++ to use the `defaultRoute` from settings, prepended with the app basePath

beac26f651e